### PR TITLE
Add code notebooks for labs to 2021 summer school

### PR DESCRIPTION
**Pull request by frankharkins**

### CODE REVIEW

**Review by eddybrando — DISMISSED** [review]
LGTM! Thanks!

File: notebooks/summer-school/2021/lab1.ipynb [hunk 1]
```diff
@@ -10,6 +10,8 @@
     "    <iframe src=\"https://www.youtube.com/embed/uI02dn7PsHI\" title=\"YouTube video player\" frameborder=\"0\" allow=\"accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture\" allowfullscreen></iframe>\n",
     "</div>\n",
     "\n",
+    "- Download the notebook: <a href=\"/content/summer-school/2021/resources/lab-notebooks/lab-1.ipynb\">[en]</a> <a href=\"/content/summer-school/2021/resources/lecture-notes/lab-1-ja.ipynb\">[ja]</a>\n",
```
Comment (vabarbosa) [inline notebooks/summer-school/2021/lab1.ipynb:13]: the URL for this Japanese notebook (`/content/summer-school/2021/resources/lecture-notes/lab-1-ja.ipynb`) appears to be incorrect. it believe it should be `/content/summer-school/2021/resources/lab-notebooks/lab-1-ja.ipynb`.
notice `lecture-notes` vs `lab-notebooks`

**Review by vabarbosa — APPROVED** [review]
@frankharkins this looks good. just one minor comment. feel free to merge after you resolve it. thanks!

File: notebooks/summer-school/2021/lab1.ipynb (hunk 1 above)
Comment (frankharkins) [inline notebooks/summer-school/2021/lab1.ipynb:13]:
```suggestion
    "- Download the notebook: <a href=\"/content/summer-school/2021/resources/lab-notebooks/lab-1.ipynb\">[en]</a> <a href=\"/content/summer-school/2021/resources/lab-notebooks/lab-1-ja.ipynb\">[ja]</a>\n",
```

File: notebooks/summer-school/2021/lab2.ipynb [hunk 2]
```diff
@@ -10,7 +10,10 @@
     "    <iframe src=\"https://www.youtube.com/embed/bNbdZ4SSjtE\" title=\"YouTube video player\" frameborder=\"0\" allow=\"accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture\" allowfullscreen></iframe>\n",
     "</div>\n",
     "\n",
+    "- Download the noteboo: : <a href=\"/content/summer-school/2021/resources/lab-notebooks/lab-2.ipynb\">[en]</a> <a href=\"/content/summer-school/2021/resources/lab-notebooks/lab-2-ja.ipynb\">[ja]</a>\n",
```
Comment (vabarbosa) [inline notebooks/summer-school/2021/lab2.ipynb:13]:
```suggestion
    "- Download the notebook: <a href=\"/content/summer-school/2021/resources/lab-notebooks/lab-2.ipynb\">[en]</a> <a href=\"/content/summer-school/2021/resources/lab-notebooks/lab-2-ja.ipynb\">[ja]</a>\n",
```

**Comment by vabarbosa** [comment]
@frankharkins thank you

**Pull request merged**